backend/app/services/adapters/team_kinds.py

```
@@ -2142,5 +2142,172 @@ def get_team_skills(
+        """Return the first available name: base_name → base_name (2) → base_name (3) ..."""
+        candidate = base_name
+        counter = 2
+        while True:
+            existing = (
+                db.query(Kind)
+                .filter(
+                    Kind.kind == kind,
+                    Kind.name == candidate,
+                    Kind.namespace == namespace,
+                    Kind.user_id == user_id,
+                    Kind.is_active == True,
```

Ruff (0.15.12)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)